core/src/main/java/io/servicecomb/core/metrics/InvocationFinishedEvent.java

```diff
@@ -29,6 +29,18 @@
 
   private final String operationType;
```

Comment:
in core project, we can use InvocationType

Reply:
Fixed